pkg/cmd/node/node_list.go

```diff
@@ -26,50 +34,75 @@ func newCmdNodeList(ctx api.Context) *cobra.Command {
 			client := mesosClient()
 
 			// The following code allows us to do the API calls for
```

Comment:
Should we use this PR to clean up the `RunE()´? Or do we revisit it when reviewing the node command? I think it is too long.